src/wallet/wallet.cpp

```diff
@@ -4316,7 +4316,7 @@ util::Result<MigrationResult> MigrateLegacyToDescriptor(std::shared_ptr<CWallet>
         }
     }
     if (!success) {
-        // Migration failed, cleanup
+        // Either migration failed or loading of the migrated wallets failed, cleanup either way
```

Comment:
In 2b8b658308c8fbfd9d41d8eb7b9491b615c43872:
Based on the commit description, this change seems more like you reasoning about the code flow than something that benefits the reader? I'm not sure it's adding much value.

Reply:
Dropped the commit.